src/loadDOM.js

```diff
@@ -0,0 +1,56 @@
+const loadDOM = () => {
+    let body  = document.querySelector(".forecast-table");
+
+    body.innerHTML =    `<div class="container">
```

Comment:
This whole html section should be in the html file as we don't do anything dynamic here. And we should use javascript only to show/hide it when needed.